clients/da-vinci-client/src/main/java/com/linkedin/davinci/blobtransfer/client/P2PFileTransferClientHandler.java

```diff
@@ -99,7 +99,11 @@ protected void channelRead0(ChannelHandlerContext ctx, HttpObject msg) throws Ex
         throw new VeniceException("No file name specified in the response for " + payload.getFullResourceName());
       }
 
-      LOGGER.debug("Starting blob transfer for file: {}", fileName);
+      LOGGER.info(
+          "Starting blob file receiving for file: {} for topic {} partition {}",
```

Comment:
Could you use Utils.getReplicaId() instead for better tracking and searching experience in log system?